packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -1955,6 +1955,191 @@ describe('State Machine handler tests', () => {
       method: 'handleCallHold',
     });
   });
+
+  describe('Call event timers tests', () => {
+    let callManager;
+    beforeEach(() => {
+      jest.useFakeTimers();
+      callManager = getCallManager(webex, defaultServiceIndicator);
+    });
+
+    afterEach(() => {
+      jest.clearAllTimers();
+    });
+
+    it('call state is set and timers are set when we receive call state', async () => {
+      const statusPayload = <WebexRequestPayload>(<unknown>{
+        statusCode: 200,
+        body: mockStatusBody,
+      });
+      const dummyEvent = {
+        type: 'E_SEND_CALL_SETUP',
+        data: undefined as any,
+      };
+      const setTimeoutForStateSpy = jest.spyOn(call as any, 'setTimeoutForState');
+      const clearCallEventsTimeoutSpy = jest.spyOn(call as any, 'clearCallEventsTimeout');
+
+      webex.request.mockReturnValue(statusPayload);
+
+      // handleOutgoingCallSetup is asynchronous
+      await call.sendCallStateMachineEvt(dummyEvent as CallEvent);
+      expect(call['callStateMachine'].state.value).toBe('S_SEND_CALL_SETUP');
+      expect(setTimeoutForStateSpy).toHaveBeenCalledWith(20000);
+      expect(clearCallEventsTimeoutSpy).not.toHaveBeenCalledTimes(1);
+
+      clearCallEventsTimeoutSpy.mockClear();
```

Comment:
Why are we clearing this mock?

Reply:
I have removed it. It was added as a precaution.